main.py

```diff
@@ -1,11 +1,377 @@
 import requests
+from customer import Customer
+from video import Video
+from rental import Rental
+from datetime import datetime, timedelta, date
+from displays import *
 
 URL = "http://127.0.0.1:5000"
 BACKUP_URL = "https://retro-video-store-api.herokuapp.com"
 
+def display(statement):
+    print(statement)
+
 def main():
-    print("WELCOME TO RETRO VIDEO STORE")
-    pass
+    display(TITLE)
+    run_cli()
+
+def list_options():
+    options = {
+        "1": "To manage customers, type 1",
+        "2": "To manage videos, type 2",
+        "3": "To manage rentals, type 3",
+        "4": "To quit, type 4"
+    }
+    display(OPTION_LINE)
+    print("Please choose from the following menu options:\n")
+
+    for choice_num in options:
+        print(f"Option {choice_num}. {options[choice_num]}")
+
+    display(OPTION_LINE)
+    return options
+
+def make_choices(options):
+    valid_choices = options.keys()
+    choice = None
+
+    while choice not in valid_choices:
+        display(LINE)
+        print("What would you like to do? ")
+        choice = input("Make your selections using the option number: ")
+        # display(LINE)
+
+    return choice
+
+def customer_options():
+    options = {
+        "1": "To list all Customers, type 1",
+        "2": "To see information for one Customer, type 2",
+        "3": "To add a new Customer, type 3",
+        "4": "To edit a Customer, type 4",
+        "5": "To delete a Customer, type 5",
+        "6": "To return to the main menu, type 6"
+    }
+    display(OPTION_LINE)
+    print("Please choose from the following Customer menu options:\n")
+
+    for choice_num in options:
+        print(f"Option {choice_num}. {options[choice_num]}")
+
+    display(OPTION_LINE)
+    return options
+
+def handle_customer_choices(customer_choices, customers):
+    # List all Customers
+    if customer_choices == "1":
+        display(LINE)
+
+        for customer in customers.list_customers():
+            print(customer)
+
+    # See information for one Customer
+    elif customer_choices == "2":
+        select_by = input("Would you like to select by name or id?: ")
+
+        if select_by == "name":
+            name = input("What is the name of your customer? ")
+            customers.get_customer(name=name)
```

Comment:
This is causing a crash because the `get_customer` function requires an id as well...  I suggest default parameters?

It's important to run through the options on your app to find bugs like this.